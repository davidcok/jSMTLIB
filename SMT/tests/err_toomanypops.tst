(set-logic QF_UF)
(push 1)
(pop 3)
(exit)
