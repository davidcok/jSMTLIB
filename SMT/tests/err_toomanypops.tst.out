success
success
(pop 3)
^^^^^^^
(error "The argument to a pop command is too large: 3 vs. a maximum of 1")
success
